raw-data/PH/phoible_inventories.tsv

```diff
@@ -481,20 +481,20 @@
 				"<sʰ>"	"Loan"			
 				"ts"				
 				"t̠ʃ"				
-				"ts̰"				
-				"t̠ʃ̰"				
+				"tsʼ"				
+				"t̠ʃʼ"				
 				"<tsʰ>"	"Loan"			
 				"<t̠ʃʰ>"	"Loan"			
 				"m"		"[m̥]"	"""Syllable-finally and word-finally"""	
 				"n"		"[n̥]"	"""Syllable-finally and word-finally"""	
-				"m̰"				
-				"n̰"				
+				"ˀm"				
+				"ˀn"				
 				"l"		"[l̥]"	"""Syllable-finally and word-finally"""	
-				"l̰"				
+				"ˀl"				
 				"w"		"[w̥]"	"""Syllable-finally and word-finally"""	
 				"j"		"[j̥]"	"""Syllable-finally and word-finally"""	
-				"w̰"				
-				"j̰"				
+				"ˀw"				
+				"ˀj"				
```

Comment:
These aren't described as "pre-glottalized" so the diacritic should be on the right of the base glyph.